src/main/scala/nl/knaw/dans/api/sword2/package.scala

```diff
@@ -43,6 +43,8 @@ package object sword2 {
 
   case class BagStoreSettings(baseDir: String, baseUrl: String)
 
+  case class PropertiesResources(bagId: String, filePaths: List[String])
```

Comment:
Both.

About the other question: the [src](https://tools.ietf.org/html/rfc4287#section-4.1.3.2) attribute must contain an IRI, so in our case a URL.